### DISCUSSION TITLE
Qt error when trying to run remotely via VNC

### DISCUSSION THREAD

**Discussion by tredondo** [keepassxreboot/keepassxc#7194] [Q&A] [answered]
After failing to run on Ubuntu 18 (#7190), I've created an Ubuntu 20 VPS cloud instance, installed `xvfb`, `x11vnc`, and all [dependencies](https://github.com/keepassxreboot/keepassxc/wiki/Set-up-Build-Environment-on-Linux) (including the optional ones). I can run xeyes remotely and see the output locally with a VNC viewer.

The problem is that trying to run the `keepassxc` binary gives the following error:

    ./keepassxc: /lib/x86_64-linux-gnu/libQt5Core.so.5: version `Qt_5.15' not found (required by ./keepassxc)

How can I run KeePassXC remotely?

**Comment by droidmonkey** [comment, marked as answer]
You built keepassxc using Qt 5.15, but are trying to run it on a machine that doesn't have that version available. Please just use the binaries we provide. AppImage, Snap, or PPA.
